client/src/components/screens/PatientSubmission/index.jsx

```diff
@@ -367,6 +367,28 @@ function PatientSubmissionScreen(props) {
         batch.observations.push(createSummary(content.summaryNote));
       }
 
+      if (get(content, 'ethnicity.value') != null) {
+        const observationBuilder = new ObservationBuilder('ETH')
+          .withStatus('final')
+          .withValue({
+            coding: [{
+              system: 'http://fhir.cqgc.ferlab.bio/CodeSystem/qc-ethnicity',
+              code: content.ethnicity.value,
+              display: intl.get(`form.patientSubmission.form.ethnicity.${content.ethnicity.value}`),
+            }],
```

Comment:
Shouldn't the `withValue` take only the `content.ethnicity.value` and all the object schema/url/etc. is handle there?

Reply:
The system can be different, but I added it as param now